drivers/SmartThings/matter-media/src/init.lua

```diff
@@ -28,6 +28,13 @@ local configure_handler = function(self, device)
     capabilities.mediaPlayback.commands.play.NAME,
     capabilities.mediaPlayback.commands.pause.NAME,
     capabilities.mediaPlayback.commands.stop.NAME,
+    capabilities.mediaPlayback.commands.rewind.NAME,
```

Comment:
Fastforward and rewind are only available if the device supports the `VS` or "Variable Speed" feature. At the time the driver was written the ZAP file did not define this feature so we couldn't use it in the driver, but thanks to @cjswedes it is now available https://github.com/project-chip/connectedhomeip/blob/76d834e409f5568434837fcad42d23dd944c1d6b/src/app/zap-templates/zcl/data-model/chip/media-playback-cluster.xml#L120. 

@HunsupJung can you check for the `VS` feature flag support and only include fastforward/rewind are supported if the feature flag is supported?

Reply:
Here is an example of checking a device for feature support: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/5a78971034bf6f6e6e02a0436300b91b91311a5f/drivers/SmartThings/matter-thermostat/src/init.lua#L43